tests/nodes_test.h

```diff
@@ -296,13 +296,8 @@ void TestUastFunctionNumberError() {
   Node *root = TreeMock();
   bool ok;
   double res = UastFilterNumber(ctx, root, "concat(//*)", &ok);
-
   CU_ASSERT_FATAL(!ok);
```

Comment:
Maybe we could instead check that the returned LastError string is not empty?

Reply:
Updated to add these checks.